lib/pbench/server/api/__init__.py

```diff
@@ -21,6 +21,8 @@
 from pbench.common.exceptions import BadConfig
 from pbench.server.utils import filesize_bytes
 
+from pbench.server.api.resources.query_controllers import QueryControllers
+
 ALLOWED_EXTENSIONS = {"xz"}
 
 app = None
```

Comment:
Can this global variable be removed?

Reply:
Um, yes; but doing so revealed a few interesting artifacts I'd missed. So... good catch.